community/portworx/values.yaml

```diff
@@ -1,8 +1,9 @@
 # Please uncomment and specify values for these options as per your requirements.
 
 
-etcdEndPoint:                         # The ETCD endpoint. Should be in the format etcd:http://<your-etcd-endpoint>:2379. If there are multiple etcd endpoints they need to be ";" seperated.
-                                        # the default value is empty since it requires to be explicity set using either the --set option of -f values.yaml.
+etcdEndPoint:                      # The ETCD endpoint. Should be in the format etcd:http://<your-etcd-endpoint>:2379. 
```

Comment:
Elsewhere, it seems that `kvdb` has taken the place of this value; if that's the case, should this have been updated to be `kvdb`? If these are in fact two different fields and this one is no longer mandatory, `kvdb` should still be included as a field so that users can set it and install from the catalog; otherwise, this has to be installed from a CLI. If changes are made to what values are present here, please also be sure to update the values-metadata.yaml to reflect those changes as well.